Gemfile

```diff
@@ -22,7 +22,7 @@ group :test do
   gem 'rspec', '~> 3.3.0'
   gem 'rspec-its'
   gem 'webmock', '~> 1.21.0'
-  gem 'http', '~> 0.8.12'
-  gem 'aruba', '~> 0.7.4'
-  gem 'cucumber', '~> 2.4.0'
+  gem 'http', '~> 2.2.2'
+  gem 'aruba', '~> 0.14.3'
```

Comment:
Let's update to `0.14.4` and drop the `absolute_paths.rb` file.

Reply:
Done :D

Reply:
You were faster! :grimacing: